Portfolio/index.html

```diff
@@ -18,7 +18,7 @@ <h1>Develop</h1>
             <li>Reviews</li>
             <li>Projects</li>

```

Comment:
The text 'JAVASCRIPT' is placed outside the 'alt' attribute of the image which may impact accessibility.